ui-v2/app/components/consul-service-list/index.hbs

```diff
@@ -35,40 +38,46 @@
   {{/if}}
   </BlockSlot>
   <BlockSlot @name="details">
-    <ul>
-    {{#if (and nspace (env 'CONSUL_NSPACES_ENABLED'))}}
-      {{#if (not-eq item.Namespace nspace)}}
-        <li class="nspace">
-          {{item.Namespace}}
-        </li>
-      {{/if}}
-    {{/if}}
-      <ConsulKind @item={{item}} as |Kind|>
-        <li>
-          <Kind />
-        </li>
-      </ConsulKind>
-      <ConsulExternalSource @item={{item}} as |ExternalSource|>
-        <li>
-          <ExternalSource />
-        </li>
-      </ConsulExternalSource>
-    {{#if (not-eq item.InstanceCount 0)}}
-      <li>
-        {{format-number item.InstanceCount}} {{pluralize item.InstanceCount 'Instance' without-count=true}}
-      </li>
-    {{/if}}
-    {{#if (get proxies item.Name)}}
-      <li data-test-proxy class="proxy">
+{{#if (and nspace (env 'CONSUL_NSPACES_ENABLED'))}}
+  {{#if (not-eq item.Namespace nspace)}}
+    <dl class="nspace">
+      <dt>
+        <Tooltip>
+            Namespace
+        </Tooltip>
+      </dt>
+      <dd>
+        {{item.Namespace}}
+      </dd>
+    </dl>
+  {{/if}}
+{{/if}}
+    <ConsulKind @item={{item}} as |Kind|>
+      <Kind />
+    </ConsulKind>
+    <ConsulExternalSource @item={{item}} as |ExternalSource|>
+      <ExternalSource />
+    </ConsulExternalSource>
```

Comment:
Just incase you miss this, see above comment (both ConsulExternalSource and ConsulKind)